prime-angular-frontend/src/app/shared/components/health-auth-site-overview-container/health-auth-site-overview-container.component.html

```diff
@@ -0,0 +1,21 @@
+<ng-container *ngIf="healthAuthoritySite">
+  <app-vendor-overview [vendor]="healthAuthoritySite"
+                       [showEditRedirect]="isIncomplete"></app-vendor-overview>
+  <app-site-information-overview [siteInformation]="healthAuthoritySite"
+                                 [showEditRedirect]="isIncomplete"></app-site-information-overview>
+  <app-health-auth-care-type-overview [healthAuthCareType]="healthAuthoritySite"
```

Comment:
Accessing the `careType` in the care type overview needs a null check `?.careType`.  If you create a Health Authority Site, but only save the first page and exit then in admin visit the only partially filled out site (only vendor) then you get the infinite dialog loop.